### ISSUE TITLE
module-getter rule is being too aggressive

### ISSUE THREAD

**Issue by alfaproject** [Gillespie59/eslint-plugin-angular#446]
There was a similar issue fixed before where the `module-getter` rule was triggered when a dependency has a `config` method, and now I stumbled upon the same thing, but with a `run` method.

Example:
```js
angular.module('app', []).controller('ctrl', function(ReportService) {
  ReportService.run(); // module-getter rule is triggered here
});
```
